rclc/include/rclc/rclc.h

```diff
@@ -50,7 +50,7 @@ rclc_destroy_node(rclc_node_t * node);
 /// Creates a rclc_publisher_t, which represents a ROS Publisher.
 rclc_publisher_t *
 rclc_create_publisher(
-  const rclc_node_t * node,
+  rclc_node_t * node,
```

Comment:
Because the node have an internal list of subscribers, so that it's modified. Well, it's not the case for publishers, but I did also that for publisher for "consistency" reason.
[The same is done with rclcpp](https://github.com/ros2/rclcpp/blob/master/rclcpp/include/rclcpp/node.hpp#L127-L133), the node is not const when creating publisher/subscription.